index.js

```diff
@@ -21,7 +21,8 @@ function elevateCategory(group) {
         .filter(entry => !!entry)
         .join(',');
     })
-    .map(string => ({ category: string }));
+    .filter(category => !!category)
```

Comment:
Here we can just used the `Boolean` constructor, so we have a conciser and more functional code. `.filter(Boolean)...`.